src/components/SearchGridManualLoad.vue

```diff
@@ -209,7 +210,7 @@ export default {
     flex-wrap: wrap;
     margin: 10px;
 
-    @media screen and (min-width: 600px) {
+    @include from(600px) {
```

Comment:
I think instead of using `from (width)` you should use Bulma's [device mixins](https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L81). This way we can achieve consistence everywhere, because  we won't be specifying custom widths on different places

Reply:
Good, I will use the `$mobile` then.  There are others custom media queries in terms of `em` and `rem` that with a font-size of 16px is approximate to that same size, I will change those too. Is it okay this way?

Reply:
Yess, Its better to use like
`@include from($destop) {
    border-left: 1px solid yellow; }`